src/canisters/governance/request.converters.ts

```diff
@@ -867,6 +867,22 @@ export const toDisburseNeuronRequest = ({
     },
   });
 
+export const toMergeMaturityRequest = ({
+  neuronId,
+  percentageToMerge,
+}: {
+  neuronId: NeuronId;
+  percentageToMerge: number;
```

Comment:
do we need a check that test the percentage is between 0 and 100?

Reply:
Good question, I guess it makes sense. The error is done in the backend, but we can also do it here.

Reply:
Good call. We defer other rules to the backend too.

Reply:
Check added.

Reply:
Coolio